### PR TITLE
Update branding to 3.0.1

### PR DESCRIPTION
**Pull request by dougbu**
- aspnet/AspNetCore-Internal#3153

### CODE REVIEW

**Comment by smitpatel** [comment]
Slightly unrelated, what is going to happen to branch release/3.0.0-preview9?

File: eng/Versions.props [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <Project>
   <PropertyGroup Label="Version settings">
-    <VersionPrefix>3.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>rc2</PreReleaseVersionLabel>
+    <VersionPrefix>3.0.1</VersionPrefix>
+    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
```
Comment (bricelam) [inline eng/Versions.props]: Did we ever fix the issue with the dotnet-ef package and servicing?

Comment (dougbu) [reply eng/Versions.props]: Sorry, which issue?

**Comment by dougbu** [comment]
> what is going to happen to branch release/3.0.0-preview9?

I may get to this cleanup tonight 😃

File: eng/Versions.props (hunk 1 above)
Comment (bricelam) [reply eng/Versions.props]: One of the directories or manifest files had “-servicing” in it and caused the tool to fail. I’ll check if it’s fixed using the internal of this commit

Comment (bricelam) [reply eng/Versions.props]: Looks like it was being caused by an SDK issue that has since been resolved: https://github.com/dotnet/sdk/pull/2739

**Pull request merged**